tools/sf-dependency-list/test/shared-flows/add-response-fapi-interaction-id/sharedflowbundle/policies/AM-Generate-New-Fapi-Interaction-Id.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2022 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AM-Generate-New-Fapi-Interaction-Id">
```

Comment:
Let's remove all unnecessary boilerplate like `async="false" continueOnError="false" enabled="true"`

and the empty tags

Reply:
done